Packs/AzureStorageTable/Integrations/AzureStorageTable/AzureStorageTable.yml

```diff
@@ -0,0 +1,278 @@
+      description: Table Name.
+      type: String
+  - arguments:
+    - default: false
+      description: Table name.
+      isArray: false
+      name: table_name
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for the partition within a given table.
+      isArray: false
+      name: partition_key
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for an entity within a given partition.
+      isArray: false
+      name: row_key
+      required: true
+      secret: false
+    - default: false
+      description: 'Entity records in JSON format: { "Key1": Value1, "Key2": Value2}'
+      isArray: false
+      name: entity_fields
+      required: true
+      secret: false
+    deprecated: false
+    description: Update an existing entity in a table.The Update Entity command does
+      not replace the existing entity.
+    execution: false
+    name: azure-storage-table-entity-update
+  - arguments:
+    - default: false
+      description: Table name.
+      isArray: false
+      name: table_name
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for the partition within a given table.
+      isArray: false
+      name: partition_key
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for an entity within a given partition.
+      isArray: false
+      name: row_key
+      required: true
+      secret: false
+    - default: false
+      description: 'Entity records in JSON format: { "Key1": Value1, "Key2": Value2}'
+      isArray: false
+      name: entity_fields
+      required: true
+      secret: false
+    deprecated: false
+    description: Replace an existing entity in a table.The Replace Entity command
+      replace the entire entity and can be used to remove properties.
+    execution: false
+    name: azure-storage-table-entity-replace
+  - arguments:
+    - default: false
+      description: Table name.
+      isArray: false
+      name: table_name
+      required: true
+      secret: false
+    - default: false
+      description: Unique identifier for the partition within a given table.
+      isArray: false
+      name: partition_key
+      required: false
+      secret: false
+    - default: false
+      description: Unique identifier for an entity within a given partition.
+      isArray: false
+      name: row_key
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Filter Entities query expression.
+        Information about Query expression structure can be found here: https://docs.microsoft.com/en-us/rest/api/storageservices/querying-tables-and-entities#constructing-filter-strings
+      isArray: false
+      name: filter
+      required: false
+      secret: false
+    - default: false
+      description: Comma-separated Entity properties to return.
+      isArray: false
```

Comment:
Shouldn't this be true? am I missing anything?

Reply:
No, it is passed as a string to the request. For example:
/myaccount/Customers()?$select=PartitionKey,RowKey,Address,CustomerSince

Reply:
should still be `isArray: true`, or it will execute the command multiple times

Reply:
Done